components/TabbedView/TabbedView.jsx

```diff
@@ -113,8 +120,8 @@ class TabbedView extends React.Component {
               key={title}
               onClick={() => this.onTabClick(title)}
               skin={skin}
-              id={`${title}-tab`}
-              aria-controls={`${title}-panel`}
+              id={`${this.sanitize(title)}-tab`}
```

Comment:
What do you think of using this [module](https://github.com/catho/quantum/blob/master/components/shared/uniqId.js) to generate the tab id?
[Here](https://github.com/catho/quantum/blob/master/components/Input/Input.jsx#L128) is an example of how to use it.